MudBlazor.Forms/Internal/MudFormsStringPropertyInput.razor

```diff
@@ -78,6 +80,26 @@ else
 
     private string? _fieldNotes;
 
+    private bool showPassword = false;
+    InputType PasswordInput = InputType.Password;
+    string PasswordInputIcon = Icons.Material.Filled.VisibilityOff;
+
+    void ToggleViewPassword()
```

Comment:
Could you add an example in the demo project that shows how you use the Password feature? Also wondering if that one should be covered in annotations + APIs

Reply:
The screenshots from the description were taken from the demo project. The toggle will be added to any password fields following the same API from before. Here's another screenshot from the demo project page showing the related attribute:

![image](https://user-images.githubusercontent.com/60167780/202473872-ae5bc7de-0dc1-45bf-862e-d73ffc308312.png)

Is that a good example or did you have something else in mind?